stubs/Modules/Base/database/migrations/2024_01_26_134541_create_open_graphs_table.php

```diff
@@ -21,7 +22,7 @@ public function up(): void
             $table->morphs('ogable');
 
             $table->string('title')->nullable();
-            $table->string('image')->nullable();
+            $table->foreignIdFor(Media::class, 'image_id')->nullable()->constrained('media');
```

Comment:
Wat vind jij?
Moeten we OG en SEO niet gewoon samenvoegen in 1 Model en table? Is eigenlijk onderdeel van hetzelfde toch? en je wilt toch altijd beide toevoegen, nooit 1 van de 2